tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -321,7 +321,7 @@ jobs:
     displayName: 'Generate cmake config'
     inputs:
       scriptPath: '$(Build.SourcesDirectory)\tools\ci_build\build.py'
-      arguments: '--config $(BuildConfig) --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --update --cmake_generator "Visual Studio 16 2019" --build_wheel --disable_contrib_ops --enable_msvc_static_runtime --enable_onnx_tests --x86'
+      arguments: '--config $(BuildConfig) --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --update --cmake_generator "Visual Studio 16 2019" --build_wheel --disable_contrib_ops --disable_ml_ops --enable_msvc_static_runtime --enable_onnx_tests --x86'
```

Comment:
What's the reason for disabling the ML ops in so many pipelines?

Reply:
It's required wherever contrib ops are disabled since that's the build config Edge uses. We've so many as some are used for gating PRs and some are used for generating the release binaries. Would be good to clean this up at some point.